README.md

```diff
@@ -1,7 +1,9 @@
 # deno
 
-Linux [![Linux](https://travis-ci.com/denoland/deno.svg?branch=master)](https://travis-ci.com/denoland/deno)
-Window [![Windows](https://ci.appveyor.com/api/projects/status/cc7nd82n34xo4nym/branch/master?svg=true)](https://ci.appveyor.com/project/ry91293/deno/branch/master)
+| **Linux** | **Windows** |
+|:---------------:|:-----------:|
+| [![Travis](https://travis-ci.com/denoland/deno.svg?branch=master)](https://travis-ci.com/denoland/deno) | [![Appveyor](https://ci.appveyor.com/api/projects/status/yel7wtcqwoy0to8x?svg=true)](https://ci.appveyor.com/project/deno/deno) |
```

Comment:
Use `?branch=master&svg=true` for appveyor, otherwise the build will randomly show as failing (as it does now).